pkgs/development/libraries/aribb25/default.nix

```
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitLab, fetchpatch, autoreconfHook, pkgconfig, pcsclite
```

`nixpkgs-fmt` suggests splitting this one per line since it's already multiple lines:
```suggestion
{ stdenv
, fetchFromGitLab
, fetchpatch
, autoreconfHook
, pkgconfig
, pcsclite
```